src/replaykit.cs

```diff
@@ -203,9 +204,12 @@ interface RPBroadcastConfiguration : NSCoding, NSSecureCoding {
 		[Export ("clipDuration")]
 		double ClipDuration { get; set; }
 
-		// TODO review, once API documentation is available, if we can provide an easier to use strongdictionary
 		[NullAllowed, Export ("videoCompressionProperties", ArgumentSemantic.Strong)]
-		NSDictionary<NSNumber, INSSecureCoding> VideoCompressionProperties { get; set; }
+#if XAMCORE_2_0
```

Comment:
This was introduced in iOS 10, so if we can get it in before the stable release it shouldn't be a breaking change.